services/fhir.ts

```diff
@@ -79,33 +85,100 @@ function getInactiveSearchParam(resourceType: string) {
     return {};
 }
 
+export async function createFHIRResource(
+    resource: AidboxResource,
+    searchParams?: SearchParams
+): Promise<RemoteDataResult<any>> {
+    return service(create(resource, searchParams));
+}
+
+export function create(resource: AidboxResource, searchParams?: SearchParams): AxiosRequestConfig {
+    return {
+        method: 'POST',
+        url: `/${resource.resourceType}`,
+        params: searchParams,
+        data: resource,
+    };
+}
+
+export async function updateFHIRResource<R extends AidboxResource>(
+    resource: AidboxReference<R>,
+    searchParams?: SearchParams
+): Promise<RemoteDataResult<R>> {
+    return service(update(resource, searchParams));
+}
+
+export function update(resource: AidboxResource, searchParams?: SearchParams): AxiosRequestConfig {
+    if (searchParams) {
+        return { method: 'PUT', url: `/${resource.resourceType}`, data: resource, params: searchParams };
+    }
+
+    if (resource.id) {
+        const versionId = resource.meta && resource.meta.versionId;
+
+        return {
+            method: 'PUT',
+            url: `/${resource.resourceType}/${resource.id}`,
+            data: resource,
+            ...(versionId ? { headers: { 'If-Match': versionId } } : {}),
+        };
+    }
+
+    throw new Error('Resourse id and search parameters are not specified');
+}
+
 export async function getFHIRResource<R extends AidboxResource>(
-    reference: AidboxReference<R>
+    reference: AidboxReference<R>,
+    searchParams?: SearchParams
 ): Promise<RemoteDataResult<R>> {
-    return service({
+    return service(get(reference, searchParams));
+}
+
+export function get<R extends AidboxResource>(
+    reference: AidboxReference<R>,
+    searchParams?: SearchParams
+): AxiosRequestConfig {
+    return {
         method: 'GET',
         url: `/${reference.resourceType}/${reference.id}`,
-    });
+        ...(searchParams ? { params: searchParams } : {}),
```

Comment:
remove. It is impossible to make get with search params because we have reference here